### DISCUSSION TITLE
How much VRAM do I need?

### DISCUSSION THREAD

**Discussion by NoMansPC** [gitmylo/audio-webui#102] [Q&A]
I think I've seen somewhere that I need like 16gb VRAM to run this properly. Did this change recently?

**Comment by gitmylo** [comment]
Depends on what you want to do. You never needed to have a lot of ram though. I don't even have 16gb, so I'm not sure who told you that.

For bark, i've gotten it to run on 3 gigabytes of vram, and you might be able to go lower since things were added for even lower usage, such as half precision.
For RVC, i'm not sure, but it isn't very high either
For AudioLDM you will need a bit more, i think like 8 gigabytes, and there's smaller models
For AudioCraft i'm not sure, but it's less than 12 gigabytes, and there's smaller models
For Whisper it's from 1-10gb, depending on which model you use.

**Comment by NoMansPC** [comment]
Sweet, thanks. I have only 6gb VRAM for now, so that's why I was wondering if it was even going to work.
